src/Elastic.Apm/Logging/LogValuesFormatter.cs

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Not sure why this got removed... maybe auto format.

Do we need this Copyright header? I don't have strong opinion on this, I'm ok with adding it back.

Reply:
This did originate from: https://github.com/aspnet/Extensions/blob/master/src/Logging/Logging.Abstractions/src/LogValuesFormatter.cs

Under Apache2 we are free to create derivatives but we have to retain this attribution. 

See also e.g: https://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications

Reply:
Oh, I see, thank you, I'll add it back.